src/metabase/search/postgres/scoring.clj

```diff
@@ -21,9 +23,12 @@
    [:inline 1]
    [:/
     [:coalesce column [:inline 0]]
-    (if (number? ceiling)
-      [:inline (double ceiling)]
-      [:cast ceiling :float])]])
+    ;; protect against div / 0
+    [:greatest
```

Comment:
hmm, does this change the math? I mean, what if the ceiling is negative? or ceiling will always be positive?

Reply:
the ranker isn't meant to be used with negative numbers, which would cause scores outside the 0-1 range.